app/Services/ModuleService.php

```diff
@@ -70,4 +81,139 @@ public function getAdminLinks(): array
     {
         return self::$adminLinks;
     }
+
+    public function createModule($array): bool
```

Comment:
Rename this to `uploadModule` and then move the `addModule` method above this. Because it's a little confusing between what create and add do. You want to be explicit and also include in the phpDocs exactly what's happening here